pages/api/giftVote/[giftVoteId].js

```diff
@@ -0,0 +1,51 @@
+import { ObjectId } from "mongodb";
+import { client } from "../../../utils/database";
+
+export default async function handler(request, response) {
+  try {
+    await client.connect();
+    const db = client.db("gifts");
+    const collection = db.collection("giftvote");
+
+    const _id = ObjectId(request.query.giftVoteId);
+
+    switch (request.method) {
+      case "GET":
+        const voteId = await collection.findOne({ _id });
+        response.status(200).json(voteId);
+
+        break;
+
+      case "PUT":
+        const voteObject = await collection.findOne({ _id });
+        const vote = voteObject.vote;
+        const { voterName, idOfClickedGift } = request.body;
+
+        const voteWithVoteCount = vote.map((gift) => {
+          if (gift.id === idOfClickedGift) {
+            gift.votes += 1;
+            gift.voterNames.push(voterName);
+          } else {
+            // gift.votes;
+          }
```

Comment:
you can get rid of the empty else branch
```suggestion
          }
```

Reply:
i removed the empty bracket